config/v2/resolvers.go

```diff
@@ -175,6 +176,24 @@ func ResolveTfLint(commons ...Common) v1.TfLint {
 	}
 }
 
+func ResolveAtlantis(commons ...Common) Atlantis {
```

Comment:
should it be `*Atlantis`? pattern matching against the other resolvers

Reply:
yeah, good catch